corehq/motech/repeaters/expression/forms.py

```diff
@@ -10,6 +10,12 @@
 from corehq.motech.repeaters.forms import GenericRepeaterForm
 
 
+LABELS = {
+    'case_action_filter_expression': _("Response case update filter expression"),
```

Comment:
nit: Response case _action_ filter expression?
Same for Response case _action_ expression?